### PR TITLE
Add BTSettl + Munari Spectral Libraries

### PR DESCRIPTION
**Pull request by lcjohnso**
Continuing work on Issue https://github.com/BEAST-Fitting/beast/issues/26, here I add BT-Settl and Munari spectral library capabilities to the code.

Questions for @mfouesneau et al.:
1) What tests can we run to check my work w.r.t. converting and packaging these spectra?  
2) What plots/info are most helpful for comparing the available libraries?

### CODE REVIEW

**Comment by mfouesneau** [comment]
Where are the library files?

**Comment by mfouesneau** [comment]
Maybe some plots like I have in the original stellib package?
http://mfouesneau.github.io/docs/pystellibs/

**Comment by lcjohnso** [comment]
Library files are (temporarily) available here: https://cass.ucsd.edu/~lcj/beast/

File: beast/physicsmodel/stars/stellib.py [hunk 1]
```diff
@@ -31,9 +31,13 @@
     'elodie_3.1': __ROOT__ + '/libs/Elodie_v3.1.grid.fits',
     'kurucz': __ROOT__ + '/libs/kurucz2004.grid.fits',
     'tlusty': __ROOT__ + '/libs/tlusty.lowres.grid.fits'
```
Comment (karllark) [inline beast/physicsmodel/stars/stellib.py]: Missing a ',' that causes an error.  See the testing results at:
https://travis-ci.org/BEAST-Fitting/beast/builds/289351508?utm_source=github_status&utm_medium=notification

**Comment by karllark** [comment]
I realized I could add this comma directly here on github using the edit on github feature.  I have done this and hopefully this will remove the error.

**Comment by karllark** [comment]
Travis failure understood - due to astropy development version.

**Pull request merged**